db/migrations/0012_alter_player_guild_alter_player_race.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-11-18 17:39
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0011_alter_skill_race'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='guilds', to='db.guild'),
```

Comment:
The `related_name` for the `guild` ForeignKey in the `Player` model is set to `'guilds'`. Consider using a more unique and descriptive `related_name` to avoid potential conflicts and improve clarity.